ffi_ioctl_32.go

```diff
@@ -1,13 +1,13 @@
 package gpio
 
-type Op uint
+import "unsafe"
 
-// Likely these constants are not portable, sorry, I don't know better.
-const (
-	GPIO_GET_CHIPINFO_IOCTL          Op = 0x8044b401
-	GPIO_GET_LINEINFO_IOCTL          Op = 0xc048b402
-	GPIO_GET_LINEHANDLE_IOCTL        Op = 0xc16cb403
-	GPIO_GET_LINEEVENT_IOCTL         Op = 0xc030b404
-	GPIOHANDLE_GET_LINE_VALUES_IOCTL Op = 0xc040b408
-	GPIOHANDLE_SET_LINE_VALUES_IOCTL Op = 0xc040b409
+// From <include/uapi/linux/gpio.h
```

Comment:
I'm afraid of possible problems with these values changed. Now that you added test that verifies them against ioR, maybe it's fine to revert to const?

Reply:
Either way it will get caught in tests so up to you

Reply:
I mean with `var GPIO_GET_LINEEVENT_IOCTL` it's possible to write external code like this: `gpio.GPIO_GET_LINEEVENT_IOCTL = 1234`

Please change them to const.